validation/src/test/scala/hmda/validation/engine/lar/quality/LarQualityEngineSpec.scala

```diff
@@ -0,0 +1,21 @@
+package hmda.validation.engine.lar.quality
+
+import hmda.parser.fi.lar.LarGenerators
+import org.scalatest.{ MustMatchers, PropSpec }
+import org.scalatest.prop.PropertyChecks
+
+class LarQualityEngineSpec
+    extends PropSpec
+    with PropertyChecks
+    with MustMatchers
+    with LarGenerators
+    with LarQualityEngine {
+
+  property("A LAR must pass quality checks") {
+    forAll(larGen) { lar =>
+      whenever(lar.id == 2) {
+        checkQuality(lar).isSuccess mustBe true
+      }
+    }
+  }
```

Comment:
I think that approach will make the generators overly complex. (I also think this question has come up before, but I don't have time to dig up the thread right now.)